simple_ddl_parser/dialects/sql.py

```diff
@@ -1606,9 +1638,22 @@ def p_expression_primary_key(self, p):
         p[0] = p[1]
 
     def p_uniq(self, p: List) -> None:
-        """uniq : UNIQUE LP pid RP"""
+        """uniq : UNIQUE LP pid RP
+        | UNIQUE KEY id LP pid RP
```

Comment:
before `UNIQUE KEY` statements were causing the parser to (silently) error and the table to not be generated.